packages/ds-healthcare-gov/src/components/Header/Header.tsx

```diff
@@ -210,7 +210,7 @@ export const Header = (props: HeaderProps) => {
 
   return (
     <>
-      {props.showUsaBanner && <UsaBanner />}
+      {props.showUsaBanner && <UsaBanner id="hc-c-header__usa-banner" />}
```

Comment:
hgov header has a convention established where all `id` are hard-coded and not dynamic. i figured since the usa banner should only appear on a page once, and with the header component, following this convention made more sense.